tests/functional/modules/test_zos_lineinfile_func.py

```diff
@@ -627,7 +631,7 @@ def test_ds_line_insertafter_regex(ansible_zos_module, dstype):
         "state":"present"
     }
     ds_name = get_tmp_ds_name()
-    temp_file = "/tmp/" + ds_name
+    temp_file = get_random_file_name(dir=TMP_DIRECTORY)
```

Comment:
if i'm not mistaken, this file won't be cleaned up in the finally block

Reply:
On set ds_eviroment function is delete the temp_file
```
def set_ds_environment(ansible_zos_module, temp_file, ds_name, ds_type, content):
    hosts = ansible_zos_module
    hosts.all.shell(cmd=f"echo \"{content}\" > {temp_file}")
    hosts.all.zos_data_set(name=ds_name, type=ds_type)
    if ds_type in ["pds", "pdse"]:
        ds_full_name = ds_name + "(MEM)"
        hosts.all.zos_data_set(name=ds_full_name, state="present", type="member")
        cmd_str = f"cp -CM {quote(temp_file)} \"//'{ds_full_name}'\""
    else:
        ds_full_name = ds_name
        cmd_str = f"cp {quote(temp_file)} \"//'{ds_full_name}'\" "
    hosts.all.shell(cmd=cmd_str)
    hosts.all.shell(cmd="rm -rf " + temp_file)
    return ds_full_name
```